### PR TITLE
Request to Update Webredirect

### PR DESCRIPTION
**Pull request by fabianosantosnet**
Hi. I made some updates in the project:
- create files: interactions.js and style.css;
- update code: index.php and homepage.html;
- improve the page style homepage.html.

I guess that are some issues that we have to do like internationalization. I suggest that the file homepage.html should be php file not html.

### CODE REVIEW

**Comment by kauffj** [comment]
Thank you @fabianosantosnet !

I've asked @nikooo777 to review.

We'd also be happy to send you some LBC as thanks, please see [this article](https://lbry.io/faq/tips).

I'm for internationalization, but I'm not sure the right way to do this is to put both on the same page.

Maybe you could create two different landing pages, one in Spanish and one in English?

File: web/homepage.html [hunk 1]
```diff
@@ -1,41 +1,29 @@
...
-      line-height: 1.5;
-    }
-    #name {
-      font-size: 24px;
-      margin-bottom: 20px;
-    }
-  </style>
+
+  <meta charset="utf-8" />  
+  <title>Open in LBRY &sol; Abrir em LBRY</title>
```
Comment (nikooo777) [inline web/homepage.html]: Yes I would suggest we don't mix 2 languages on the same page. Could you change this one back to English only?

File: web/homepage.html [hunk 2]
```diff
@@ -1,41 +1,29 @@
...
+  <link rel="stylesheet" href="style.css" />
+
 </head>
+
 <body>
-  <h1>Open with LBRY</h1>
+
+ <div id="wrapper">
+ 
+  <h1>Open with LBRY / Abrir em LBRY</h1>
```
Comment (nikooo777) [inline web/homepage.html]: Same here

File: web/homepage.html [hunk 3]
```diff
@@ -1,41 +1,29 @@
...
   <div>
-    <input type="text" id="name" placeholder="name" />
+    <input type="text" id="name" placeholder="Search &sol; Buscar" />
```
Comment (nikooo777) [inline web/homepage.html]: and here

**Review by nikooo777 — CHANGES_REQUESTED** [review]
Thank you, the refactor is good. If you fix those 3 lines I'll proceed with merging :)

**Comment by fabianosantosnet** [comment]
Hi. I made the changes.

**Review by nikooo777 — APPROVED** [review]

**Comment by nikooo777** [comment]
Thank you @fabianosantosnet!
As @kauffj don't hesitate to take a look at the article above, maybe join us on discord so we can offer you a tip for your help :)

**Comment by tzarebczan** [comment]
@nikooo777 the prompt where you enter URIs now says "search" - it should say something along the lines of "Enter LBRY Claim / Channel here". Also, before it used to show that the lbry:// is prepended, now it's not clear (unless they look at the resulting URL)

**Pull request merged**